libr/asm/p/asm_arm_hacks.inc

```diff
@@ -0,0 +1,222 @@
+static int hack_handle_dp_imm(ut32 insn, char **buf_asm) {
+	char *mnemonic = NULL;
+	const ut8 op0 = (insn >> 23) & 0x7;
+
+	// Add/subtract (immediate, with tags)
+	if (op0 == 3) {
+		const ut8 sf = (insn >> 31) & 0x1;
+		const ut8 op = (insn >> 30) & 0x1;
+		const ut8 S = (insn >> 29) & 0x1;
+		const ut8 o2 = (insn >> 2) & 0x1;
+		if (sf == 1 && op == 0 && S == 0 && o2 == 0) {
+			mnemonic = "addg";
+		} else if (sf == 1 && op == 1 && S == 0 && o2 == 0) {
+			mnemonic = "subg";
+		} else {
+			return -1;
+		}
+		const ut8 uimm6 = ((insn >> 16) & 0x3f) << 4;
+		const ut8 uimm4 = (insn >> 10) & 0xf;
+		const ut8 Xn = (insn >> 5) & 0x1f;
+		const ut8 Xd = (insn >> 0) & 0x1f;
+		*buf_asm = r_str_newf ("%s x%d, x%d, #0x%x, #0x%x",
```

Comment:
i'm kinda sure capstone does not use as value format `#0x1234` but `0x1234`

Reply:
The following code https://gist.github.com/phakeobj/8b7d50bfeff4e0f510899a42519ffe32 outputs:
```
$ clang test1.c -o test1 -lcapstone
$ ./test1
0x1000:	mov		w9, #0
```

Could you please elaborate?

Reply:
The following code https://gist.github.com/phakeobj/f287fe2b83018c3ee004795f2178e3be
```
$ clang test1.c -o test1 -lcapstone
$ ./test1
0x1000:	ldr		x0, [x0, #0x100]
```

Could you please elaborate?

Reply:
Could you?